R/split_SE_components.R

```diff
@@ -52,7 +52,7 @@ split_SE_components <- function(df_, nested_keys = NULL, combine_on = 1L) {
   identifiers_md <- get_env_identifiers(simplify = TRUE)
   identifiers_md$nested_keys <- nested_keys
 
-  df_ <- S4Vectors::DataFrame(df_)
+  df_ <- S4Vectors::DataFrame(df_, check.names = FALSE)
```

Comment:
Probably, we should add test for data with colname with special character - to check if really everything is calculated as expected :)